kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/build/revapi-config.json

```diff
@@ -28,7 +28,278 @@
+                   "annotationType": "io.swagger.annotations.ApiOperation",
+                   "annotation": "@io.swagger.annotations.ApiOperation(\"Queries process tasks by variables\")",
+                   "package": "org.kie.server.remote.rest.jbpm",
+                   "classSimpleName": "RuntimeDataResource",
+                   "methodName": "queryUserTasksByVariables",
+                   "elementKind": "method",
+                   "justification": "JBPM-9296 Self contained Audit/History module"
+                 },
+                 {
+                   "code": "java.annotation.added",
+                   "old": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryUserTasksByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer)",
+                   "new": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryUserTasksByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer)",
+                   "annotationType": "org.kie.server.remote.rest.common.marker.KieServerEndpoint",
+                   "annotation": "@org.kie.server.remote.rest.common.marker.KieServerEndpoint(categories = {org.kie.server.remote.rest.common.marker.KieServerEndpoint.EndpointType.DEFAULT, org.kie.server.remote.rest.common.marker.KieServerEndpoint.EndpointType.HISTORY})",
+                   "package": "org.kie.server.remote.rest.jbpm",
+                   "classSimpleName": "RuntimeDataResource",
+                   "methodName": "queryUserTasksByVariables",
+                   "elementKind": "method",
+                   "justification": "JBPM-9296 Self contained Audit/History module"
+                 },
+                 {
+                   "code": "java.annotation.added",
+                   "old": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryProcessesByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer)",
+                   "new": "method javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource::queryProcessesByVariables(javax.ws.rs.core.HttpHeaders, java.lang.String, java.lang.Integer, java.lang.Integer)",
+                   "annotationType": "io.swagger.annotations.ApiOperation",
+                   "annotation": "@io.swagger.annotations.ApiOperation(\"Queries processes by variables and tasks\")",
+                   "package": "org.kie.server.remote.rest.jbpm",
+                   "classSimpleName": "RuntimeDataResource",
+                   "methodName": "queryProcessesByVariables",
+                   "elementKind": "method",
+                   "justification": "JBPM-9296 Self contained Audit/History module"
+                 },
```

Comment:
This ignore section is not needed, changes are already in 7.44. final version. Please remove.
```suggestion
```

Reply:
removed.